core/loadpoint.go

```diff
@@ -279,43 +279,36 @@ func NewLoadpoint(log *util.Logger, settings settings.Settings) *Loadpoint {
 func (lp *Loadpoint) migrateSettings() {
 	// One-time migrations MUST be mirrored in restoreSettings
 	if lp.DefaultMode != "" {
-		lp.log.WARN.Println("deprecated: mode setting is ignored, please remove")
 		if _, err := lp.settings.String(keys.Mode); err != nil {
 			lp.settings.SetString(keys.Mode, string(lp.DefaultMode))
 		}
 	}
 	if lp.Title_ != "" {
-		lp.log.WARN.Println("deprecated: title setting is ignored, please remove")
 		if _, err := lp.settings.String(keys.Title); err != nil {
```

Comment:
Mir erschließt sich nicht wieso wir den Titel in die DB übertragen. Das führt bei yaml-konfigurierten LPs dazu, dass der Text aus der yaml in die DB Settings geschrieben wird und von dem Zeitpunkt an quasi nicht mehr änderbar ist. Hier haben wir glaube ich noch einen Denkfehler. Gleiches gilt für Soc.Estimate. Für yaml-LPs sollten die weiterhin einzig aus der YAML kommen.

Reply:
@naltatis lass uns bitte einmal im Detail drauf schauen.